File: SEP3-TIER1/BlazorTest/Communication/Fields.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace BlazorTest.Model
{
    public class Fields
    {
        [JsonProperty("user", NullValueHandling = NullValueHandling.Ignore)]
        public User User { get; set; }

        [JsonProperty("projects", NullValueHandling = NullValueHandling.Ignore)]
        public ICollection<Project> Projects { get; set; }

        [JsonProperty("pendinginvitation", NullValueHandling = NullValueHandling.Ignore)]
        public PendingInvitation PendingInvitation { get; set; }

        [JsonProperty("channels", NullValueHandling = NullValueHandling.Ignore)]
        public ICollection<Channel> Channels { get; set; }

        [JsonProperty("channelusers", NullValueHandling = NullValueHandling.Ignore)]
        public ICollection<ChannelUser> ChannelUsers { get; set; }

        [JsonProperty("projectchannels", NullValueHandling = NullValueHandling.Ignore)]
        public List<ProjectChannel> ProjectChannels { get; set; }

        [JsonProperty("channelmessages", NullValueHandling = NullValueHandling.Ignore)]

        public List<CMessage> ChannelMessages { get; set; }

        [JsonProperty("comments", NullValueHandling = NullValueHandling.Ignore)]
        public ICollection<TaskComment> Comments { get; set; }

        [JsonProperty("tasks", NullValueHandling = NullValueHandling.Ignore)]
        public ICollection<Task> Tasks { get; set; }

        [JsonProperty("subtasks", NullValueHandling = NullValueHandling.Ignore)]
        public ICollection<SubTask> SubTasks { get; set; }

        [JsonProperty("password", NullValueHandling = NullValueHandling.Ignore)]
        public string Password { get; set; }

        [JsonProperty("secondpassword", NullValueHandling = NullValueHandling.Ignore)]
        public string SecondPassword { get; set; }

        [JsonProperty("responsecode", NullValueHandling = NullValueHandling.Ignore)]
        public int ResponseCode { get; set; }

        [JsonProperty("responsedescription", NullValueHandling = NullValueHandling.Ignore)]
        public string ResponseDescription { get; set; }
    }
}

File: SEP3-TIER2/src/Server/Model/Communication/Fields.java
package Server.Model.Communication;


import Server.Model.*;

import java.util.List;

public class Fields {

    private User user;
    private List<Project> projects;
    private PendingInvitation pendinginvitation;
    private List<Task> tasks;
    private List<SubTask> subtasks;
    private List<TaskComment> comments;
    private List<Channel> channels;
    private List<ChannelUser> channelusers;
    private List<CMessage> channelmessages;
    private String password;
    private int responsecode;
    private String responsedescription;

    public Fields() { }

    public User getUser() {
        return user;
    }

    public void setUser(User user) {
        this.user = user;
    }

    public List<Project> getProjects() {
        return projects;
    }

    public void setProjects(List<Project> projects) {
        this.projects = projects;
    }

    public PendingInvitation getPendinginvitation() {
        return pendinginvitation;
    }

    public void setPendinginvitation(PendingInvitation pendinginvitation) {
        this.pendinginvitation = pendinginvitation;
    }

    public int getResponsecode() {
        return responsecode;
    }

    public void setResponsecode(int responsecode) {
        this.responsecode = responsecode;
    }

    public String getResponsedescription() {
        return responsedescription;
    }

    public void setResponsedescription(String responsedescription) {
        this.responsedescription = responsedescription;
    }

    public List<Task> getTasks() {
        return tasks;
    }

    public void setTasks(List<Task> tasks) {
        this.tasks = tasks;
    }

    public List<TaskComment> getComments() {
        return comments;
    }

    public void setComments(List<TaskComment> comments) {
        this.comments = comments;
    }

    public List<Channel> getChannels() {
        return channels;
    }

    public void setChannels(List<Channel> channels) {
        this.channels = channels;
    }

    public List<ChannelUser> getChannelusers() {
        return channelusers;
    }

    public void setChannelusers(List<ChannelUser> channelusers) {
        this.channelusers = channelusers;
    }

    public List<CMessage> getChannelmessages() {
        return channelmessages;
    }

    public void setChannelmessages(List<CMessage> channelmessages) {
        this.channelmessages = channelmessages;
    }

    public List<SubTask> getSubtasks() {
        return subtasks;
    }

    public void setSubtasks(List<SubTask> subtasks) {
        this.subtasks = subtasks;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    @Override
    public String toString() {
        return "Fields{" +
                "user=" + user +
                ", projects=" + projects +
                ", pendinginvitation=" + pendinginvitation +
                ", tasks=" + tasks +
                ", subtasks=" + subtasks +
                ", comments=" + comments +
                ", channels=" + channels +
                ", channelusers=" + channelusers +
                ", channelmessages=" + channelmessages +
                ", password='" + password + '\'' +
                ", responsecode=" + responsecode +
                ", responsedescription='" + responsedescription + '\'' +
                '}';
    }
}
